### PR TITLE
SRCH-5351: frontend setup for vote.gov link

### PR DESCRIPTION
**Pull request by yogesh27**
## Summary
- frontend setup for vote.gov link
 
### Checklist
Please ensure you have addressed all concerns below before marking a PR "ready for review" or before requesting a re-review. If you cannot complete an item below, replace the checkbox with the ⚠️ `:warning:` emoji and explain why the step was not completed.
 
#### Functionality Checks
 
- [x] You have merged the latest changes from the target branch (usually `main`) into your branch.
  
- [x] Your primary commit message is of the format **SRCH-#### \<description\>** matching the associated Jira ticket.

- [x] PR title is either of the format **SRCH-#### \<description\>** matching the associated Jira ticket (i.e. "SRCH-123 implement feature X"), or **Release - SRCH-####, SRCH-####, SRCH-####** matching the Jira ticket numbers in the release.
 
- [ ] Automated checks pass. If Code Climate checks do not pass, explain reason for failures:
 
#### Process Checks

- [x] You have specified at least one "Reviewer".

### CODE REVIEW

File: app/javascript/components/Identifier/Identifier.tsx [hunk 1]
```diff
@@ -42,7 +43,7 @@ const StyledUswdsIdentifier = styled(UswdsIdentifier).attrs<{ styles: FontsAndCo
 `;
 
 // eslint-disable-next-line complexity
-export const Identifier = ({ identifierContent, identifierLinks }: IdentifierProps) => {
+export const Identifier = ({ identifierContent, identifierLinks, showVoteOrgLink }: IdentifierProps) => {
```
Comment (stevenbarragan) [inline app/javascript/components/Identifier/Identifier.tsx:46]: Can we add a `false` to `showVoteOrgLink` as a default so we don't have to pass it all the time.

**Review by stevenbarragan — CHANGES_REQUESTED** [review]

File: app/javascript/components/Identifier/Identifier.tsx (hunk 1 above)
Comment (yogesh27) [reply app/javascript/components/Identifier/Identifier.tsx]: PR is updated.

File: app/javascript/components/Identifier/Identifier.tsx [hunk 2]
```diff
@@ -93,6 +94,17 @@ export const Identifier = ({ identifierContent, identifierLinks }: IdentifierPro
             </Link>
           </IdentifierGov>
         )}
+        {showVoteOrgLink && (
+          <IdentifierGov aria-label="U.S. government information and services" className="padding-bottom-0 padding-top-0">
```
[the remark this answers is not in the document]
Comment (yogesh27) [reply app/javascript/components/Identifier/Identifier.tsx]: PR is updated.

**Review by stevenbarragan — APPROVED** [review]
:shipit:

**Review by stevenbarragan — APPROVED** [review]

**Pull request merged**